packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketSelectedOfferVerify.tsx

```diff
@@ -0,0 +1,204 @@
+import styled from 'styled-components';
+import addressValidator from 'trezor-address-validator';
+import { QuestionTooltip, Translation } from 'src/components/suite';
+import { Input, variables, Button } from '@trezor/components';
+import { AddressOptions } from 'src/views/wallet/coinmarket/common';
+import { useTranslation } from 'src/hooks/suite/useTranslation';
+import { ConfirmedOnTrezor } from 'src/views/wallet/coinmarket/common/ConfirmedOnTrezor';
+import { cryptoToCoinSymbol } from 'src/utils/wallet/coinmarket/cryptoSymbolUtils';
+import { useCoinmarketFormContext } from 'src/hooks/wallet/coinmarket/form/useCoinmarketCommonForm';
+import CoinmarketSelectedOfferVerifyOptions from './CoinmarketSelectedOfferVerifyOptions';
+import { CoinmarketTradeBuyType } from 'src/types/coinmarket/coinmarket';
+import useCoinmarketVerifyAccount from 'src/hooks/wallet/coinmarket/form/useCoinmarketVerifyAccount';
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-top: 10px;
+`;
+
+const Heading = styled.div`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+    padding: 16px 24px 0;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    font-size: ${variables.FONT_SIZE.SMALL};
+`;
+
+const CardContent = styled.div`
+    display: flex;
+    flex-direction: column;
+    padding: 0 24px;
+`;
+
+const Label = styled.div`
+    display: flex;
+    align-items: center;
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const StyledQuestionTooltip = styled(QuestionTooltip)`
+    padding-left: 3px;
+`;
+
+const CustomLabel = styled(Label)`
+    padding: 12px 0;
+`;
+
+const ButtonWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-top: 20px;
+    border-top: 1px solid ${({ theme }) => theme.STROKE_GREY};
+    margin: 20px 0;
+`;
+
+const Row = styled.div`
+    margin: 12px 0;
+`;
+
+const CoinmarketSelectedOfferVerify = () => {
```

Comment:
Please consider to handover selected quote over props. Please check if the if(!currency) return null us needed.

Reply:
Looks like for `BuyTrade` `receiveCurrency` can be `undefined`... not necessary to add to props (need to be controlled anyway.)